frontend/src/features/admin-form/create/CreatePageDrawerContext.tsx

```diff
@@ -26,29 +29,29 @@ type BuilderDrawerContextProps = {
   handleLogicClick: () => void
 }
 
-const BuilderDrawerContext = createContext<
-  BuilderDrawerContextProps | undefined
+const CreatePageDrawerContext = createContext<
+  CreatePageDrawerContextProps | undefined
 >(undefined)
 
 /**
  * Provider component that makes drawer context object available to any
  * child component that calls `useBuilderDrawer()`.
```

Comment:
should we update this to `useCreatePageDrawer`?

Reply:
[804d3fc](https://github.com/opengovsg/FormSG/pull/3481/commits/804d3fc36b12deceaceade133d4697bba7e1c71f)